bullet-featherstone/src/EntityManagement_TEST.cc

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright (C) 2022 Open Source Robotics Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+*/
+
+#include <gtest/gtest.h>
+
+#include <gz/plugin/Loader.hh>
+
+#include <gz/math/eigen3/Conversions.hh>
+
+#include <gz/physics/RequestEngine.hh>
+
+#include "EntityManagementFeatures.hh"
+#include "JointFeatures.hh"
+
+// ToDo(Lobotuerk): Once more tests are added into this plugin, delete line 35
```

Comment:
can we remove this comment ? and maybe the test ? I think this should be covered by the common test